EventListener/TemplateListener.php

```diff
@@ -103,12 +103,12 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             };
 
             $event->setResponse(new StreamedResponse($callback));
+        } else {
+            $event->setResponse($templating->renderResponse($template->getTemplate(), $parameters));
         }
 
         // make sure the owner (controller+dependencies) is not cached or stored elsewhere
         $template->setOwner(array());
```

Comment:
Are you sure this doesn't have to be run before `$event->setResponse()` like before?